src/elements/content-sidebar/activity-feed/annotation-thread/__tests__/AnnotationThread.test.js

```diff
@@ -1,18 +1,28 @@
 // @flow
 import { render } from '@testing-library/react';
 import React from 'react';
+import { IntlProvider } from 'react-intl';
 import AnnotationThread from '../AnnotationThread';
 
+jest.mock('react-intl', () => jest.requireActual('react-intl'));
+jest.mock('../AnnotationThreadContent', () => () => <div data-testid="annotation-content" />);
+jest.mock('../AnnotationThreadCreate', () => () => <div data-testid="annotation-create" />);
+
 describe('elements/content-sidebar/activity-feed/annotation-thread/AnnotationThread', () => {
-    const getWrapper = (props = {}) => render(<AnnotationThread {...props} />);
+    const IntlWrapper = ({ children }: { children?: React.ReactNode }) => {
+        return <IntlProvider locale="en">{children}</IntlProvider>;
+    };
+    const getWrapper = props => render(<AnnotationThread {...props} />, { wrapper: IntlWrapper });
 
-    test('should not render if annotationId is undefined', async () => {
-        const { container } = getWrapper();
-        expect(container).toBeEmptyDOMElement();
+    test('should render AnnotationThreadCreate if annotationId is undefined', async () => {
+        const { getByTestId } = getWrapper();
+        expect(getByTestId('annotation-thread')).toBeInTheDocument();
```

Comment:
Should there be an assertion that the `AnnotationThreadContent` is present or not present in these tests since `AnnotationThread` is just the overall component?